ChangeLog.d/ecp-light.txt

```diff
@@ -0,0 +1,8 @@
+Features
```

Comment:
I think this commit being included is a rebasing accident, you probably want to `rebase -i` to remove it.

Reply:
Yes, absolutely, there are at least 2 commits which are totally wrong. Sorry, í'll rebase again removing them